.github/workflows/cmd.yml

```diff
@@ -0,0 +1,323 @@
+name: Command
+
+on:
+  issue_comment: # listen for comments on issues
+    types: [created]
+
+permissions: # allow the action to comment on the PR
+  contents: write
+  issues: write
+  pull-requests: write
+  actions: read
+
+jobs:
+  acknowledge:
+    if: ${{ startsWith(github.event.comment.body, '/cmd') }}
```

Comment:
anyone can trigger this?

Reply:
yep good point, will add a similar check as for /merge
and cmd.md guide
thanks